src/Internal.AspNetCore.Sdk/Internal.AspNetCore.Sdk.csproj

```diff
@@ -4,9 +4,8 @@
 
   <PropertyGroup>
     <VersionPrefix>2.1.0</VersionPrefix>
-    <TargetFramework>netstandard1.0</TargetFramework>
```

Comment:
Keep at netstandard1.0. We need this package to be compatible with all TFMs. We still compile some netstandard1.x projects and this would break that.